open_group.py

```diff
@@ -0,0 +1,14 @@
+from selenium import webdriver
```

Comment:
Hey, I think you misunderstood me.

I meant to add a method in the existing Facebook class so that once the user has logged in, he can navigate to different groups he enters.

Can you please move this method to the already existing class? That's be great.

Reply:
okk, Should it to be moved in same file(fb_login.py)?